src/server_manager/model/gcp.ts

```diff
@@ -12,14 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {ManagedServer, RegionId} from './server';
+import {DataCenterMap} from './location';
+import {ManagedServer} from './server';
 
 // Keys are region IDs like "us-central1".
 // Values are zones like ["us-central1-a", "us-central1-b"].
 export type ZoneId = string;
```

Comment:
Zone would probably benefit from being a class, so it can provide location() and region() methods, avoiding the anemic model. But it's fine as is.

Reply:
I've added a `Zone` class implementing `CloudLocation`.